README.md

```diff
@@ -16,20 +16,32 @@ Zeebe provides visibility into and control over business processes that span mul
 * Export process data for monitoring and analysis
 * Engage with an active community
 
-[Learn more at zeebe.io](https://zeebe.io)
+[Learn more at camunda.com](https://camunda.com/products/cloud/workflow-engine/)
+
+## Release Lifecycle
+
+Our release cadence within major releases is a minor release every three months, with an alpha release on each of the two months between minor releases. Releases happen on the second Tuesday of the month, Berlin time.
+
+Minor releases are supported with patches for six months after their release.
+
+Here is a diagram illustrating the lifecycle of minor releases over a 13-month period:
+
+![](img/release-cadence.png)
```

Comment:
There is a typo (i.e. `x.2-alpha1` should be `x.2-alpha2`) in this image. Also, it might help to just use text (since the image is just a picture of text)

Reply:
Thanks for the catch. I'll fix that. 

I made it an image to avoid overflow and get resizing on different screen sizes.